generated/nidaqmx/nidaqmx_service.cpp

```diff
@@ -1161,7 +1161,7 @@ namespace nidaqmx_grpc {
     try {
       auto task_grpc_session = request->task();
       TaskHandle task = session_repository_->access_session(task_grpc_session.id(), task_grpc_session.name());
-      session_repository_->remove_session(task);
+      session_repository_->remove_session(task_grpc_session.id(), task_grpc_session.name());
```

Comment:
Why the change?
`remove_session` wasn't working for the RFmx services. There is no correct mapping from RFmx handle to grpc session. This is because if two sessions are "initialized" on the same device it will be one RFmx handle but two grpc_device sessions.

Why is it safe?
In each case, we have the grpc_session as input so we can just use those in a similar way to `access_session` to figure out the session_id to clear out of the `session_repository` and `session_resource_repository`.